styles/helpers/typography.css

```diff
@@ -30,6 +30,10 @@ body {
   text-align: right;
 }
 
+.text-stretch {
+  @mixin text-stretch 2px;
```

Comment:
feels like the 2px should be a var driven by the config and then sites could override the yaml var if they wanted to tweak it